--- fribidi-0.19.7/lib/debug.h.omv~	2016-03-21 00:32:14.736615854 +0100
+++ fribidi-0.19.7/lib/debug.h	2016-03-21 00:33:26.796169357 +0100
@@ -60,15 +60,15 @@
 #ifndef MSG
 #define MSG(s) \
 	FRIBIDI_BEGIN_STMT \
-	FRIBIDI_FPRINTF(FRIBIDI_STDERR_ s); \
+	FRIBIDI_FPRINTF(stderr, s); \
 	FRIBIDI_END_STMT
 #define MSG2(s, t) \
 	FRIBIDI_BEGIN_STMT \
-	FRIBIDI_FPRINTF(FRIBIDI_STDERR_ s, t); \
+	FRIBIDI_FPRINTF(stderr, s, t); \
 	FRIBIDI_END_STMT
 #define MSG5(s, t, u, v, w) \
 	FRIBIDI_BEGIN_STMT \
-	FRIBIDI_FPRINTF(FRIBIDI_STDERR_ s, t, u, v, w); \
+	FRIBIDI_FPRINTF(stderr, s, t, u, v, w); \
 	FRIBIDI_END_STMT
 #endif /* !MSG */
 
